config/v1/types_console.go

```diff
@@ -26,6 +26,9 @@ type Console struct {
 type ConsoleSpec struct {
 	// +optional
 	Authentication ConsoleAuthentication `json:"authentication"`
+
+	// +optional
+	Helm HelmConfig `json:"helm"`
```

Comment:
Need doc on this field. Why would a cluster-admin set it?  I looked for a linked enhancement.

Reply:
Thanks, good question!

- From a user perspective, an admin determines which is the helm catalog that would be available for the rest of the users in the cluster.

- From a technical standpoint, the helm catalog url is a config on Console, managed by the console-operator.